### DISCUSSION TITLE
OPDS2 Stable version

### DISCUSSION THREAD

**Discussion by fillito** [opds-community/drafts#78] [General]
Is there any roadmap for making OPDS 2 stable and promoting it from Draft to 2.0.0? 

We want to add support for it in Panels. But doing it in a draft state is quite risky for us because we can't retroactively update AppStore versions if something changes. Keeping the "draft" status means that any breaking change can be introduced. 

It would be nice to have a stable version and use semantic versioning. 
Many people are already using the current draft anyway. It looks like the current version is stable enough to deserve a proper public version. 

Thoughts?

**Comment by llemeurfr** [comment]
OPDS 2 should have been tagged as stable for a long time, that is right. 
We have to decide if the details of the specification are clear enough, and if yes move it to "stable", I think.

**Comment by HadrienGardeur** [comment]
Hello @fillito,

As @llemeurfr pointed out, OPDS 2.0 has been stable and used in production for a few years by now. Most of the on-going discussions are related to extensions and/or satellite specs that shouldn't get in the way of a server/client implementing OPDS 2.0.

As with any community-led project, this is mostly a question of time/resource allocated to move OPDS from its current draft state into something a little more official.

I've been meaning to do that for a while but always lacked the time to do so. I'm now in a position where I can dedicate more of my time to this task and I'll do my best to push the various OPDS-related specifications forward.

It's hard to commit to a roadmap at this point, but I'll come up with a better estimate in the next few weeks.

**Comment by fillito** [comment]
Thank you folks! That sounds great! 

@HadrienGardeur let me know if I can assist/help with anything. I know I haven't participated much in this community, but I'm eager to do so and contribute within my capacity. 

Knowing the promotion to 2.0.0 is feasible and coming in the near future, we will start working on implementing it on Panels very soon.
